plugins/package-managers/gradle-inspector/src/funTest/assets/projects/synthetic/multi-kotlin-project-expected-output-cli.yml

```diff
@@ -42,9 +34,6 @@ project:
       - id: "Maven:org.jetbrains.kotlin:kotlin-reflect:1.7.20"
         dependencies:
         - id: "Maven:org.jetbrains.kotlin:kotlin-stdlib:1.7.20"
-          dependencies:
```

Comment:
`kotlin-stdlib`'s dependencies were already seen on a level closer to the root (see line 49 and below), so they are skipped here. I'm not sure why the `Gradle` package manager implementation does not skip it, too.

Reply:
> I'm not sure why the `Gradle` package manager implementation does not skip it, too.

That's because the "legacy" Gradle analyzer natively creates a *graph* which is then resolved back to a *tree* for test result comparison, and this expands the cycles again.